lib/checkfunctions.cpp

```diff
@@ -873,4 +874,5 @@ void CheckFunctions::getErrorMessages(ErrorLogger *errorLogger, const Settings *
     c.missingReturnError(nullptr);
     c.copyElisionError(nullptr);
     c.useStandardLibraryError(nullptr, "memcpy");
+    // TODO: allocaCalled, <funcname>Called
```

Comment:
hmm instead of `<funcname>Called` I feel it is better to use the same id and then provide the funcname in the symbol instead. But that is something we can tweak in another PR.